package exceptions;


import java.util.Arrays;
import java.util.function.Function;

public class rewiew_q {
    public static void main(String[] args) throws Exception {
//        //   fall(11);
//        //   fall(null);
//
//        //throw new Exception();
//        //throw new Exception("Ow! I fell.");
//        //throw new RuntimeException();
//        throw new RuntimeException("Ow! I fell.");
//
//    }

//    static void fall(int distance) throws Exception {
//        try {
//
//            if (distance > 10) {
//                throw new Exception(new Exception("distance > 10"));
//            }
//        } catch (Exception d) {
//            d.printStackTrace();
//        }
//
//    }
//    static  void fall(String input) {
//        System.out.println(input.toLowerCase());
  }

}